pkg/certmaker/certmaker.go

```diff
@@ -522,3 +541,47 @@ func ValidateKMSConfig(config KMSConfig) error {
 
 	return nil
 }
+
+// ToSignatureAlgorithm returns the x509.SignatureAlgorithm for the given signer and hash algorithm.
+func ToSignatureAlgorithm(signer crypto.Signer, hash crypto.Hash) (x509.SignatureAlgorithm, error) {
```

Comment:
Can you move this to https://github.com/sigstore/fulcio/blob/main/pkg/ca/common.go?

For context, `pkg/certmaker` was just added to support a new CLI utility, and I'd prefer to add new functions to it that are only used by the utility.

We should still keep the places above where you updated `tmpl.SignatureAlgorithm` though, that LGTM.